app/components/decisions/decision-view-help.vue

```diff
@@ -0,0 +1,64 @@
+<template>
+	<span style="display:inline-block;">
+		<a href="javascript:void(0)" @click="startTour">{{title}}</a>
```

Comment:
dont use `javascript:void(0)`

```suggestion
		<a href="#" @click.prevent="startTour">{{title}}</a>
```